changelog.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+- [Updated API Docs to use `mysql.driver`](https://github.com/ballerina-platform/ballerina-standard-library/issues/3710)
```

Comment:
```suggestion
- [Updated API Docs on `mysql.driver` usgaes](https://github.com/ballerina-platform/ballerina-standard-library/issues/3710)
```

Reply:
Fixed in [a9ff096](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/726/commits/a9ff0969f27883d6d959dead62ab704c7e893fa3)